infra/terraform/external_function/snowalert/kms.tf

```diff
@@ -21,7 +21,7 @@ resource "aws_kms_key" "prod" {
 }
 
 resource "aws_iam_policy" "kms_decrypt" {
-  name        = "snowalert_kms_decrypt"
+  name        = var.kms_decrypt_name
```

Comment:
I'm not sure if this one makes sense as a full variable, but the name should not mention snowalert, either. maybe something with the prefix var and describing what it does?

the latest version also uses AWS Secrets, so we'll want to include those before releasing